pilot/pkg/xds/monitoring.go

```diff
@@ -171,6 +172,14 @@ var (
 	inboundEDSUpdates     = inboundUpdates.With(typeTag.Value("eds"))
 	inboundServiceUpdates = inboundUpdates.With(typeTag.Value("svc"))
 	inboundServiceDeletes = inboundUpdates.With(typeTag.Value("svcdelete"))
+
+	configSizeBytes = monitoring.NewDistribution(
+		"pilot_xds_config_size_bytes",
+		"Distribution of configuration sizes pushed to clients",
+		[]float64{1, 1000, 10000, 100000, 500000, 1000000},
```

Comment:
Not sure those are ideal - 10k, 1M, 4M, 10M, 40M ? 
I think 4M is the default gRPC limit, 10M is where configs are probably too much, and above 40M it's very likely user will hit problems.

@howardjohn  - I think the top buckets are more critical, not sure how much we care about low buckets.

Reply:
Yeah, this is the big thing. We want to get the bucketization scheme to have resolution at the boundaries we care about.   I don't have a good idea what those boundaries are.

Should we switch to [ 1, 10000, 1000000, 4000000, 10000000, 40000000 ] ?

Reply:
SGTM